### PR TITLE
Review

### PR DESCRIPTION
**Pull request by jihaneH**

### CODE REVIEW

**Comment by clintonjosephs** [comment]
Hi @jihaneH 👋🏻,

I admire the way you structured your project. You have done an amazing job 💪🏻!

Project Highlights 👏🏻

✔️ Project was properly modulated
✔️ Good use of ES6 arrow functions
✔️ Professional README
✔️ Good interface and styling
✔️ Your code did not entertain DRY

Also, see below a few things that can be enhanced 👍

**Review by clintonjosephs — COMMENTED** [review]
Hi @jihaneH 👋🏻,

I admire the way you structured your project. You have done an amazing job 💪🏻!

Project Highlights 👏🏻

✔️ Project was properly modulated 🏅 
✔️ Good use of ES6 arrow functions🏅 
✔️ Professional README🏅 
✔️ Good interface and styling🏅 
✔️ You did not leave any commented code on your build🏅 
✔️ You have .lock file to avoid uploading node_modules files🏅 

Also, see below a few things that can be enhanced 👍

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  </head>
+  <body>
+    <section>
+      <div class="todo">
+        <div class="header">
+          <h1>Today's To Do</h1>
+          <i class="fas fa-sync"></i>
+        </div>
+        <div class="add_task"><input type="text" placeholder="add to your list" /><i class="fas fa-plus"></i></div>
+        <div class="tasks depot" id="depot" data-dropeffect="move"></div>
```
[the remark this answers is not in the document]
Comment (jihaneH) [reply src/index.html:19]: 👌, thank you

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  input.id = `checkbox_task_${task.index}`;
+  input.dataset.id = task.index;
+  input.addEventListener('change', () => {
+    updateStatus(task.index);
+  });
+  input.addEventListener('click', () => {
+    deleteTask(task.index);
+  });
+
+  const input2 = createHtml(taskInput, 'input');
```
[the remark this answers is not in the document]
Comment (jihaneH) [reply src/index.js:43]: 👌, thank you

File: src/modules/drag_and_drop.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            }
+          }
+        }
+      } else if (elToMove.dataset.position - dernierElementParcouru.dataset.position < 1) {
+        const max = dernierElementParcouru.dataset.position;
+        const min = elToMove.dataset.position;
+
+        dernierElementParcouru.after(elToMove);
+
+        for (let i = 0; i < items.length; i += 1) {
```
[the remark this answers is not in the document]
Comment (jihaneH) [reply src/modules/drag_and_drop.js:78]: 👌, thank you

**Comment by jihaneH** [comment]
> Hi @jihane,
> 
> STATUS: APPROVED 🕺🥳
> 
> Congratulations🍾🥳!! Your project is ready for presentation! There is nothing else to say other than... it's time to merge it :shipit:
> 
> Alt Text
> 
> Good luck! 🎉

Thank you! 😄

**Pull request merged**